### PR TITLE
use new UUID methods

### PR DESCRIPTION
**Pull request by myleshorton**

### CODE REVIEW

File: web/cache.go [hunk 1]
```diff
@@ -38,20 +38,26 @@ type cacheEntry []byte
 
 func (c *cache) newCacheEntry() cacheEntry {
 	ce := make([]byte, widthPermalink+encoding.WidthTime+1)
-	permalink := uuid.NewRandom()
-	copy(ce[idxPermalink:], permalink)
+	permalink := uuid.New()
+	copy(ce[idxPermalink:], permalink[:])
```
Comment (myleshorton) [inline web/cache.go]: A little weird to convert from an array to a slice to an array here...but I think the alternative is to make `cacheEntry` an array - `[16]byte`

File: web/cache.go [hunk 2]
```diff
@@ -38,20 +38,26 @@ type cacheEntry []byte
...
 
 func (ce cacheEntry) permalinkBytes() []byte {
 	return ce[idxPermalink : idxPermalink+widthPermalink]
 }
 
+func (ce cacheEntry) uuid() uuid.UUID {
+	var arr [16]byte
+	copy(arr[:], ce.permalinkBytes())
+	return uuid.UUID(arr)
+}
```
Comment (myleshorton) [inline web/cache.go]: This is the only other tricky part -- if we do this a lot it might make sense to avoid the copy if possible

Comment (oxtoacart) [reply web/cache.go]: We don't do it a lot.

**Comment by myleshorton** [comment]
So I'm pretty sure this is all correct -- just really a matter of if it affects performance.

**Comment by myleshorton** [comment]
Thanks @oxtoacart!

**Pull request merged**